### PR TITLE
{bio}[intel/2018a] CIRCexplorer2 v2.3.3, TopHat v2.1.0, Cufflinks v2.2.1, ... (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
{bio}[intel/2018a] CIRCexplorer2 v2.3.3 w/ Python 2.7.14

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node3161.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/894750caa93723a016bba228dc8bf6a7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2458.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a0e1a2ab69de16e9c2127076b90141a8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2109.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/79318a3333eee51fd4a73bbf8524eaa1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2520.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1e6d7144bb4650c5f1b6f677cb39935e for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 6 out of 12 (6 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/05aca045849997a6a8ec6a205929bb1a for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 4 out of 9 (6 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/19ce20b58c88107d53dd4f988e45bfdc for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 5 out of 7 (6 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/0e91a9cb35f1204681fe35b30046ab5b for a full test report.

**Comment by vanzod** [comment]
I cannot test it because I do not have the sources for `Kent_tools-20171107`.
If you can update the dependency with `Kent_tools-20180716` (#6606) then I can do it.

File: easybuild/easyconfigs/c/CIRCexplorer2/CIRCexplorer2-2.3.3-intel-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['30f9273e4a9ab1575a001d1165ba19972995f68d2acbfbc1b113fca49054d8ae']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('TopHat', '2.1.1', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CIRCexplorer2/CIRCexplorer2-2.3.3-intel-2018a-Python-2.7.14.eb]: I got word that using `TopHat` 2.1.1 leads to problems with `tophat-reports`.

According to https://www.biostars.org/p/186979/, the recommended workaround is to use `TopHat` 2.1.0 instead...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2107.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f4adac1d901feee1e45c07bb2a1f4e30 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 11 out of 11 (6 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/51abadb567e36e5d7c7de26db66f9132 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/b30ef6484ef91a4852ff25369f9b29e5 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**